src/data/form-interaction.js

```diff
@@ -1,51 +1,16 @@
 /*jshint esnext:true */
+/* global Utils, updateModel */
+
 (function() {
 
 'use strict';
 var table = document.querySelector('.worked-days-table');
 
-attachHighlightHandlers();
-attachUnlinkHandlers();
-attachDisplayedHandler();
 attachInputHandler();
 attachEmptyCellHandler();
 
 function attachInputHandler() {
-  table.addEventListener('input', onInput);
-  table.addEventListener('input', Utils.throttle(onThrottledInput, 500));
-}
-
-function attachHighlightHandlers() {
-  table.addEventListener('mouseover', (e) => {
-    var td;
-    if ((td = e.target.closest('td'))) {
-      highlightHolidayFromCell(td);
-    }
-  });
-
-  table.addEventListener('mouseout', (e) => {
-    if (e.target.closest('td')) {
-      removeAllHighlights();
-    }
-  });
-}
-
-function attachDisplayedHandler() {
-  window.addEventListener('table-displayed', function onTableDisplayed() {
-    window.removeEventListener('table-displayed', onTableDisplayed);
-
-    buildModel();
-  });
-}
-
-function attachUnlinkHandlers() {
-  table.addEventListener('click', (e) => {
-    if (e.target.matches('.unlink-button')) {
-      e.stopImmediatePropagation();
-      e.preventDefault();
-      handleUnlink(e.target.closest('td'));
-    }
-  });
+  table.addEventListener('input', Utils.throttle(onThrottledInput.bind(null, updateModel), 2000));
```

Comment:
The fact you moved from 500 to 2000 makes things quite slow; but I suspect it's necessary because updateModel regenerates everything always (?). Have you tried with 1000 ?